### PR TITLE
Remove unused _TypeNone enum field.

### PR DESCRIPTION
**Pull request by aam**
_TypeNone is not used and that triggers hint in dart analyzer since dart 2.7.0

### CODE REVIEW

File: lib/ui/painting.dart [hunk 1]
```diff
@@ -2633,6 +2633,7 @@ class ColorFilter {
 
   // The type of SkColorFilter class to create for Skia.
   // These constants must be kept in sync with ColorFilterType in paint.cc.
```
Comment (dnfield) [inline lib/ui/painting.dart]: This line is no longer true and should just be removed.

File: lib/ui/painting.dart [hunk 2]
```diff
@@ -2633,6 +2633,7 @@ class ColorFilter {
...
+  // ignore:unused_field
   static const int _TypeNone = 0; // null
```
Comment (dnfield) [inline lib/ui/painting.dart]: Rather than adding an ignore, let's just remove the unused field.

We should also do the same in the web_ui painting.dart

Comment (aam) [reply lib/ui/painting.dart]: There is no _TypeNone in web_ui painting.dart as far as I can tell

File: lib/ui/painting.dart (hunk 1 above)
Comment (aam) [reply lib/ui/painting.dart]: Done

File: lib/ui/painting.dart (hunk 2 above)
Comment (aam) [reply lib/ui/painting.dart]: > Rather than adding an ignore, let's just remove the unused field.

Done

Comment (dnfield) [reply lib/ui/painting.dart]: It's in `lib/web_ui/lib/src/engine/color_filter.dart`, line 118

Comment (aam) [reply lib/ui/painting.dart]: Ah, got you. Interesting this is not picked up by analyzer as unused field.

**Pull request merged**